pkg/e2e/verify/namespace_webhook.go

```diff
@@ -231,7 +231,16 @@ func updateNamespace(namespace string, asUser string, userGroup string, h *helpe
 		UserName: asUser,
 		Groups:   userGroups,
 	})
-	_, err = h.Kube().CoreV1().Namespaces().Update(context.TODO(), ns, metav1.UpdateOptions{})
+	updatedNamespace, err := h.Kube().CoreV1().Namespaces().Update(context.TODO(), ns, metav1.UpdateOptions{})
```

Comment:
should this error be checked, and short circuit the get below?

Reply:
Absolutely! Updated.